### ISSUE TITLE
the sun

### ISSUE THREAD

**Issue by CarrotSprout** [sixthsurge/photon#25]
why is the sun a menacing blue orb
![2023-06-30_10 58 13](https://github.com/sixthsurge/photon/assets/138153167/a5403301-7b91-4475-b1cd-1a69ba0e85a0)

**Comment by sixthsurge** [comment]
It looks like the tonemapping isn't working
Did you change any settings? Are there any "Invalid program" errors in the chat when you load the shader?

**Comment by Patomosm** [comment]
I don't have this issue, neither with Optifine nor with Iris.

**Comment by CarrotSprout** [comment]
i dont get any errors  in the chat, its just like this

**Comment by CarrotSprout** [comment]
i kept the settings the same too

**Comment by CarrotSprout** [comment]
im not sure if this helps but im using minecraft 1.19.4

**Comment by Patomosm** [comment]
Now I am on 1.20 but I used 1.19.4 before and I hadn't thi issue.
It may be a graphics driver error. Try updating it or wait for an update. Do you have AMD or NVIDIA GPU? With updated NVIDIA driver there is no issue to me.

**Comment by CarrotSprout** [comment]
i just got a new pc with an rtx 3070 so i dont know if the drivers for this pc are outdated

**Comment by ozialaz** [comment]
I have the same issue with you. GPU is rtx3070ti with the latest driver.

**Comment by sixthsurge** [comment]
Could you run Photon and then send me your latest.log file?

**Comment by Patomosm** [comment]
Check yout driver settings: set it to default settings (RGB corol output, not YCbCr, at the max dynamic range).
Check your monitor Color Space settings: mine is set to DCI-P3 (ST2084 Range); lower settings may limit the identification of very bright colors, such as the sun, or fluorescent colors.

**Comment by tsumi666** [comment]
Going to close this as no response. Feel free to reopen this issue with a log file

**Comment by Rocklost** [comment]
I had this issue as well running Optifine. This bug went away when I switched to Iris & Sodium (This is in no way putting down Optifine, I am still quite fond of it but for Photon I use I&S)

Alternatively, through a little tweaking, I disabled the sun, so it only appeared as an invisible light source, with no actual sun texture. I can't remember the exact settings I used, but it was something along the lines of Vanilla Sun + Disable Sun/Moon.
